server/services/index.ts

```diff
@@ -54,9 +55,16 @@ const submitDraftReportService = new SubmitDraftReportService(
 )
 
 const updateDraftReportService = new UpdateDraftReportService(draftReportClient, elite2ClientBuilder, systemToken)
+const draftInvolvedStaffService = new DraftInvolvedStaffService(
+  authClientBuilder,
+  elite2ClientBuilder,
+  draftReportClient,
+  userService
+)
 
 const draftReportService = new DraftReportService(
   draftReportClient,
+  draftInvolvedStaffService,
```

Comment:
this has loads of dependencies - could do with looking at breaking this down a bit.